### ISSUE TITLE
Add Tests for Admin Itinerary Controller (MarkIncomplete Action)

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#1041]
Add some tests to validate expected behaviour of the MarkIncomplete action within the admin ItineraryController.

**Comment by gftrader** [comment]
@tonysurma it looks like these tests were completed in pull #1042. Github is showing a note next to that commit saying this will close once #1042 is merged into the master.
